Cargo.toml

```diff
@@ -8,6 +8,7 @@ default-run = "divviup_api_bin"
 [features]
 default = []
 aggregator-api-mock = []
+kind-integration = []
```

Comment:
"Running it in Kind" is a shorthand for what we (Divvi Up) would use this for, but I think we should name this so it better captures what it does. How about `stub-api-integrations`?

Reply:
This doesn't actually stub api integrations, though. It's more like "integration test mode" — the http clients are still configured to talk to the external internet